app/queries/app/global.ts

```diff
@@ -33,8 +37,47 @@ export const observeProfileLongPresTutorial = () => {
     if (!appDatabase) {
         return of$(false);
     }
-    return appDatabase.get<GlobalModel>(GLOBAL).query(Q.where('id', GLOBAL_IDENTIFIERS.PROFILE_LONG_PRESS_TUTORIAL), Q.take(1)).observe().pipe(
+    return queryGlobalValue(appDatabase, GLOBAL_IDENTIFIERS.PROFILE_LONG_PRESS_TUTORIAL).observe().pipe(
         switchMap((result) => (result.length ? result[0].observe() : of$(false))),
         switchMap((v) => of$(Boolean(v))),
     );
 };
+
+export const getLastAskedForReview = async () => {
+    const appDatabase = DatabaseManager.appDatabase?.database;
+    if (!appDatabase) {
+        return 0;
+    }
+    const value = await queryGlobalValue(appDatabase, GLOBAL_IDENTIFIERS.LAST_ASK_FOR_REVIEW).fetch();
+    if (!value[0]?.value) {
+        return 0;
+    }
+
+    return value[0].value;
+};
+
+export const getDontAskForReview = async () => {
+    const appDatabase = DatabaseManager.appDatabase?.database;
+    if (!appDatabase) {
+        return false;
+    }
+    const value = await queryGlobalValue(appDatabase, GLOBAL_IDENTIFIERS.DONT_ASK_FOR_REVIEW).fetch();
```

Comment:
maybe rename the const to `records` so that it reads `records[0].value` 

same in the other functions